### ISSUE TITLE
Couldn't process image on BytesIO image

### ISSUE THREAD

**Issue by SaschaKrug** [aFelipeSP/pdfme#24]
Hi, I found the following issue with BytesIO images: If an image gets pushed to another page, its get called twice. At the first time, the BytesIO gets read and closed. At the second time it fails because of `I/O operation on closed file`.

As I guess you need to load the image to see if it doesn't fit and load it on another page, I assume loading it twice is required.

So maybe just add use `image_bytes = copy(image)` here:
https://github.com/aFelipeSP/pdfme/blob/759e9d38ce073f9de1ff2202cc349a0ea72e2bd6/pdfme/image.py#L46

If you like, I can provide a PR.

**Comment by aFelipeSP** [comment]
You are right, I was using an invalid type in image class to compare to incoming bytes, and I was closing the bytes stream in a different place too. I was in a rush earlier, and didn't notice it. I created a new version to address this. Let me know if you find a new issue.

**Comment by SaschaKrug** [comment]
And now it's working! Thank you for the quick support.
